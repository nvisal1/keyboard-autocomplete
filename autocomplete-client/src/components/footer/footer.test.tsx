import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './footer';

describe('When the Footer component is created', () => {
    it('should render correctly', () => {
        const currentSnapshot = renderer.create(<Footer></Footer>).toJSON();

        expect(currentSnapshot).toMatchSnapshot();
    });
});
