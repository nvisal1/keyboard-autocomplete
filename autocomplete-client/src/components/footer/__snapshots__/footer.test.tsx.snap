// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When the Footer component is created should render correctly 1`] = `
<div
  className="footer"
/>
`;
